goth_tests/pyproject.toml

```diff
@@ -23,7 +23,7 @@ pytest-asyncio = "^0.20.2"
 pytest-rerunfailures = "^10.3"
 #goth = "^0.11"
 # to use development goth version uncomment below
-goth =  { git = "https://github.com/golemfactory/goth.git", rev = "master" }
+goth =  { git = "https://github.com/golemfactory/goth.git", rev = "d2951a62e2a7cf0712f7f4a66c4a080777841611" }
```

Comment:
This change is already in master branch

Reply:
Done.

Reply:
Reverted it to commit hash, because `master` will not be updated without `poetry update`. 
Alternatively we could create new release of the goth (which means bumping version and updating changelog).